### ISSUE TITLE
Keep option for: get_cluster - return indices vs. return the data-points directly

### ISSUE THREAD

**Issue by mhfzsharmin** [annoviko/pyclustering#411]
In current implementation, cure_instance.get_clusters() returns the actuals data points. However, the description says "Returns list of allocated clusters, each cluster contains indexes of objects in list of data". Returning indexes are better than returning actual data-points. Or add an option what to return - indices or the data-point. Thanks

**Comment by annoviko** [comment]
Hello, @mhfzsharmin,

Thank you for your message. Could you please provide version of **pyclustering**? It was changed since 0.7.0 - and since that moment indexes are returned instead of actual points. I have run examples from **pyclustering/cluster/cure** and obtain following output:
```
[[7, 5, 8, 6, 9], [4, 1, 3, 0, 2]]
Sample:  D:\workspace\pyclustering\pyclustering\samples\samples\simple\Simple01.data 		Execution time:  0.0013756217026062686 

[[10, 11, 13, 12, 14], [17, 15, 16, 21, 19, 22, 18, 20], [5, 9, 1, 6, 3, 4, 0, 2, 7, 8]]
Sample:  D:\workspace\pyclustering\pyclustering\samples\samples\simple\Simple02.data 		Execution time:  0.005254162146596908 
```

Could you please update pyclustering version?

**Comment by mhfzsharmin** [comment]
Thanks for your quick reply. I noticed that it returns indices for other clustering, e.g. I have used SOM, Kmeans/kmedians and dbscan. However, for Cure clustering it is the data points not indices. My version is "0.7.2". Anyways, I will check for updated version. Thanks again!

**Comment by annoviko** [comment]
@mhfzsharmin,

Could you please share you code example? I will double check the issue again, but real example would help to clarify what is going on.

**Comment by annoviko** [comment]
Hello, @mhfzsharmin,

I have released 0.8.0 version where issue is resolved. You were right, it was changed but included to release only now, sorry for confusing with version. Release 0.8.0 is available on pypi: https://pypi.python.org/pypi/pyclustering .

**Comment by mhfzsharmin** [comment]
Many thanks for the update!
